### DISCUSSION TITLE
Ethernet Support...

### DISCUSSION THREAD

**Discussion by lpuntaloro** [HomeSpan/HomeSpan#626] [General]
Hi, I know that currently ethernet support for esp32 is not working. Is there any news? Thank you

**Comment by HomeSpan** [comment]
I have not re-explored but I noticed a few threads on the web where others have had some success getting MDSN to work over Ethernet after making various modifications to the sketch.  I'll check out and see if there is anything that can be used to robustly address the issue.

**Reply by lpuntaloro** [reply]
Hi...

Did you get to look inside? 
Thank you

**Comment by HomeSpan** [comment]
Have not had a chance but hopefully will be able to revisit.
